### DISCUSSION TITLE
elemental2 cannot be resolved to a typeJava

### DISCUSSION THREAD

**Discussion by treblereel** [salesforce/bazel-vscode-java#19] [Q&A]
First of all, thank you for work you do ! Looks like it's exactly what i need

I have a little problem using external maven dependancy with your plugin, i think, i missed something

evn:
VSCode : Version: 1.63.0 (Universal) on Mac M1
bazel-vscode v0.0.4
config:

```
{
    "java.import.bazel.enabled": true,
    "java.import.maven.enabled": false,
    "files.exclude": {
        "**/.classpath": true,
        "**/.project": true,
        "**/.settings": true,
        "**/.factorypath": true
    },
    "editor.suggestSelection": "first",
    "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
    "redhat.telemetry.enabled": true,
    "extensions.ignoreRecommendations": true
}
```
 bazel run :AppRunner
>> Hi! class elemental2.dom.DomGlobal

so bazel import works (i am new to bazel, so i could be wrong in any of the steps)

please help

[java-tutorial.zip](https://github.com/salesforce/bazel-vscode/files/7700163/java-tutorial.zip)

**Comment by guw** [comment]
We do not support all ways of working Maven dependencies yet. An you provide more details on you project?

**Comment by treblereel** [comment]
@guw thank you for your answer

i have attached the demo, but i suppose where are could be better options.
So may i ask you to share best practice ?

any option would be great for me

**Reply by guw** [reply]
Please use the reply functionality of GitHub discussions. It makes it easier to follow.

Unfortunately, we don't have cycles currently to help with debug projects. This project is in an early stage. If you are new to Bazel then you probably should reach out to Bazel mailing list or StackOverflow for recommendations.

We have a sample/demo project here that we use for testing:
https://github.com/salesforce/bazel-ls-demo-project

We do not have a full test suite.
